reference/docs-conceptual/developer/cmdlet/how-to-invoke-a-cmdlet-from-within-a-cmdlet.md

```diff
@@ -89,7 +85,7 @@ namespace SendGreeting
       gp.Name = new string[] { "[a-t]*" };
       foreach (Process p in gp.Invoke<Process>())
       {
-        Console.WriteLine(p.ToString());
+        WriteVerbose(p.ToString());
```

Comment:
We probably don't want to encourage `Console.WriteLine` from a PowerShell command

Reply:
Amen